### DISCUSSION TITLE
News - Robert Schumann Hochschule Düsseldorf

### DISCUSSION THREAD

**Discussion by reinhardwh** [jamulussoftware/jamulus#2578] [General]
Robert Schumann Hochschule Düsseldorf use Jamulus

https://www.rsh-duesseldorf.de/institute/institut-fuer-kirchenmusik/jamulus/

**Comment by ann0see** [comment]
Interesting to know ;-). The "Deutsche Chorverband" also featured us, I think? Not sure though. https://www.saechsischer-chorverband.de/scv/medien/news/jumu.html at least did. 

Edit: https://www.youtube.com/watch?v=Z9BJhhi1zEQ

**Reply by reinhardwh** [reply]
Yes, a good article :)

**Comment by PeRAux** [comment]
It is also used in Lübeck (with a little help from a friend ;-))

https://www.fbs-luebeck.de/blog/blog-details/auszeit-fuer-eltern-neues-angebot-der-musikschule-der-gemeinnuetzigen/

And i heard that the "Bosch Big Band" also used Jamulus.
Musicians from Bosch Big Band also spread it to others 
https://www.dhv-bw.de/veranstaltungen/online-proben-mit-jamulus-teil-1-technische-grundlagen/

I'd like to add my own "thank you" here!
I stumbled across Jamulus in 2020 - and it brought together old band members during hard times!
Thanks a lot for all the good work!!
